package.json

```diff
@@ -4,11 +4,15 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
+    "bcrypt": "^4.0.1",
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
```

Comment:
Please make ur .gitmodules not include `package.lock.json` and `package.json` in code reviews!